### PR TITLE
CLR-024 viewListClass-editClass

### PR DESCRIPTION
**Pull request by nhanlinh1478**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "rules": {
     "prettier/prettier": ["error"],
     "no-unused-vars": [
-      "warn",
+      "warning",
```
Comment (nguyen315) [inline .eslintrc]:
```suggestion
      "warn",
```

File: src/Routes.js [hunk 2]
```diff
@@ -15,9 +15,14 @@ import Profile from './pages/Profile'
...
 import NotHavePermission from 'src/components/NotHavePermission'
 import ManageAdmins from 'src/pages/ManageAdmins'
+<<<<<<< HEAD
 import CreateAdmin from 'src/pages/CreateAdmin'
 import AdminDetails from 'src/pages/AdminDetails'
 
+=======
+import ManageUsers from './pages/ManageClasses'
+import UserList from './pages/UserList'
+>>>>>>> 33de060 (viewListClass-editClass)
```
Comment (nguyen315) [inline src/Routes.js]: Resolve conflict in this file first

File: src/pages/ManageClasses/index.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  const [confirmDialog, setConfirmDialog] = useState({
+    isOpen: false,
+    title: '',
+    subTitle: '',
+  })
+  const [idClassroom, setIdClassroom] = useState()
+  let history = useHistory()
+  useEffect(() => {
+    async function FetchClassrooms() {
+      const result = await axiosClient.get('/api/classrooms/all-classrooms')
```
Comment (nguyen315) [inline src/pages/ManageClasses/index.js]: Change endpoint here to match with new endpoint at API

File: src/pages/ManageClasses/index.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  const [classrooms, setClassrooms] = useState()
+  const [confirmDialog, setConfirmDialog] = useState({
+    isOpen: false,
+    title: '',
+    subTitle: '',
+  })
+  const [idClassroom, setIdClassroom] = useState()
+  let history = useHistory()
+  useEffect(() => {
+    async function FetchClassrooms() {
```
Comment (nguyen315) [inline src/pages/ManageClasses/index.js]:
```suggestion
    async function fetchClassrooms() {
```

File: src/pages/ManageClasses/index.js [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+      const editTime = result.data.map((classroom) => {
+        return {
+          ...classroom,
+          updatedAt: format(new Date(classroom.updatedAt), 'MMM,dd,yyyy'),
+        }
+      })
+      setClassrooms(editTime)
+    }
+    return FetchClassrooms()
```
Comment (nguyen315) [inline src/pages/ManageClasses/index.js]:
```suggestion
    return fetchClassrooms()
```

File: src/pages/ManageClasses/index.js [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+  }, [])
+  const handleDeleClasses = () => {}
```
Comment (nguyen315) [inline src/pages/ManageClasses/index.js]: Why this function blank?

File: src/pages/ManageClasses/index.js [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+              }}
+            >
+              <DeleteIcon />
+            </Button>
+          </>
+        )
+      },
+    },
+  ]
+  console.log('idClass:', idClassroom)
```
Comment (nguyen315) [inline src/pages/ManageClasses/index.js]: remove console

**Review by nguyen315 — COMMENTED** [review]
minor fix

File: src/pages/ManageClasses/index.js (hunk 6 above)
Comment (nhanlinh1478) [reply src/pages/ManageClasses/index.js]: not use

Comment (nhanlinh1478) [reply src/pages/ManageClasses/index.js]: deleted

**Pull request merged**